src/components/interfaces/MOBILE_API.xml

```diff
@@ -3102,6 +3164,252 @@
                 A set of all ventilation modes that are controllable.
             </description>
         </param>
+        <param name="heatedSteeringWheelAvailable" type="Boolean" mandatory="false">
+          <description>
+            Availability of the control (enable/disable) of heated Steering Wheel.
+            True: Available, False: Not Available, Not present: Not Available.
+          </description>
+        </param>
+        <param name="heatedWindshieldAvailable" type="Boolean" mandatory="false">
+          <description>
+            Availability of the control (enable/disable) of heated Windshield.
+            True: Available, False: Not Available, Not present: Not Available.
+          </description>
+        </param>
+        <param name="heatedRearWindowAvailable" type="Boolean" mandatory="false">
+          <description>
+            Availability of the control (enable/disable) of heated Rear Window.
+            True: Available, False: Not Available, Not present: Not Available.
+          </description>
+        </param>
+        <param name="heatedMirrorsAvailable" type="Boolean" mandatory="false">
+          <description>
+            Availability of the control (enable/disable) of heated Mirrors.
+            True: Available, False: Not Available, Not present: Not Available.
+          </description>
+        </param>
+    </struct>
+
+    <struct name="EqualizerSettings">
+        <description>Defines the each Equalizer channel settings.</description>
+        <param name="channelId" type="Integer" minvalue="1" maxvalue="100" mandatory="true"></param>
+        <param name="channelName" type="String" mandatory="false" maxlength="50">
+           <description>read-only channel / frequency name (e.i. "Treble, Midrange, Bass" or "125 Hz")</description>
+        </param>
+        <param name="channelSetting" type="Integer" minvalue="0" maxvalue="100" mandatory="true"></param>
+    </struct>
+
+    <struct name="AudioControlData">
+        <param name="source" type="PrimaryAudioSource" mandatory="false">
+            <description>
+            In a getter response or a notification, it is the current primary audio source of the system.
+            In a setter request, it is the target audio source that the system shall switch to.
+            If the value is MOBILE_APP, the system shall switch to the mobile media app that issues the setter RPC.
+        </description>
```

Comment:
Indentation is incorrect

Reply:
@jacobkeeler Fixed at commit 593f38dd0a9cfb336623c77fa160eb6f871f806e